elements/pl-symbolic-input/pl-symbolic-input.mustache

```diff
@@ -67,21 +85,29 @@ ${{a_tru}}$
 {{/answer}}
 
 {{#format}}
-Your answer must be a symbolic expression. All numbers must be rational - so, <code>1/2</code> instead of <code>0.5</code>.
-{{^allow_complex}}Complex numbers are not allowed.{{/allow_complex}}
-<hr>
+<p>Your answer must be a symbolic expression. All numbers must be rational - so, <code>1/2</code> instead of <code>0.5</code>.
```

Comment:
Should we be using `class="user-output"` everywhere? The default red code stands out to me like it's an error.

Reply:
Woops yes, meant to change this.